Formula/g2s.rb

```diff
@@ -0,0 +1,50 @@
+class G2s < Formula
+  desc "Toolbox for geostatistical simulations"
+  homepage "https://gaia-unil.github.io/G2S/"
+  url "https://github.com/GAIA-UNIL/G2S/archive/refs/tags/v0.98.015.tar.gz"
+  sha256 "d6074c6c904640e3c69c4f8236e4d8cb3061d4d0c6234437b1803111d33e49d1"
+  license "GPL-3.0-only"
+
+  # Add dependencies
+  depends_on "cppzmq"
+  depends_on "curl"
+  depends_on "fftw"
+  depends_on "jsoncpp"
+  depends_on "libomp"
+  depends_on "zeromq"
+  depends_on "zlib"
+
+  def install
+    cd "build" do
+      # Run "make c++ -j"
+      system "make", "c++", "-j", "CXXFLAGS=-Xclang -fopenmp -I#{Formula["fftw"].opt_include}", \
+          "LIB_PATH=-L#{Formula["fftw"].opt_lib} -L#{Formula["libomp"].opt_lib} -lomp"
```

Comment:
Can it not determine these paths from a tool like pkg-config?
```suggestion
      ENV["CXXFLAGS"] = "-Xclang -fopenmp -I#{Formula["fftw"].opt_include}"
      ENV["LIB_PATH"] = "-L#{Formula["fftw"].opt_lib} -L#{Formula["libomp"].opt_lib} -lomp"
      system "make", "c++"
```